docs/guides/main.rst

```diff
@@ -132,6 +132,19 @@ trouble or you have more detail to contribute here, please direct it to
 .. _install pip: https://pip.pypa.io/en/stable/installing/
 .. _get-pip.py: https://bootstrap.pypa.io/get-pip.py
 
+Installing on ARM (Raspberry Pi and similar)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Beets on ARM devices is not recommended for Linux novices. If you are
+comfortable with light troubleshooting in tools like ``pip``, ``make``,
+and beets' command-line binary dependencies (e.g. `ffmpeg` and
+`ImageMagick`), you will probably be okay on ARM devices like the
```

Comment:
Didn't notice `ffmpeg` and `ImageMagick` initially: see how single and double backticks end up being formatted in the docs:

```rst
  Such a re-import can be done with, in this case, `beet import -L data_source:=MusicBrainz`.
* :doc:`plugins/autobpm`: Fix the ``TypeError`` where tempo was being returned
```
![image](https://github.com/user-attachments/assets/16ee7c80-b855-4de2-be80-c717fd11e873)

If your intention is to have them in italic, then keep it as it is, otherwise you can add double backticks